_drafts/submitting-your-first-pull-request.markdown

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: Submitting Your First Pull Request
+author: dave_paquette
+---
+
```

Comment:
Suggestion: Have a table at the beginning or end listing the commands you used here with a short description of each. E.g.

| `git checkout -b NewBranch` | Creates a new branch and changes to it |
| `git add -A` | Stages your changes for committing |